src/url-replacements.js

```diff
@@ -239,6 +239,14 @@ class UrlReplacements {
       return this.getTimingData_('navigationStart',
           'domContentLoadedEventStart');
     });
+
+    // Returns an identifier for the viewer.
+    this.set_('VIEWER', () => {
+      return viewerFor(this.win_).getViewerOrigin().then(origin => {
+        const url = parseUrl(origin);
+        return (url && url.hostname) || '';
```

Comment:
@rudygalfi  what do you think? i am fine with that. that also works if in future, viewerOrigin stops being URL (for example for apps.)